scripts/test/system.sh

```diff
@@ -187,7 +188,50 @@ _subctl cloud prepare generic --context cluster1
 # Deprecated variant
 _subctl cloud prepare generic --kubecontext cluster1 && exit 1
 
+# Rename old broker-info.subm to be able to compare it with the recovered broker-info.sum
+echo "Renaming broker-info.sum to broker-info.subm.orig"
+mv "${DAPPER_SOURCE}"/output/broker-info.subm "${DAPPER_SOURCE}"/output/broker-info.subm.orig
+
+# all blues
+# Test subctl recover-broker-info invocations which will always fail (no matter the broker and Submariner installations and the current
+# context set) with broker not found error
+_subctl recover-broker-info --namespace non-existent && exit 1 #column 2
+_subctl recover-broker-info --namespace submariner-k8s-broker --brokercontext cluster2 && exit 1 #column 4
+_subctl recover-broker-info --context cluster2 && exit 1 #column 7
```

Comment:
What's the namespace used here? Do we default `--namespace` to _submariner-k8s-broker_?

Reply:
yes. Default namespace with `--context` is `subamriner-operator` and with `--brokercontext` is ` submariner-k8s-broker`